components/PerformanceHighlight/PerformanceHighlight.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Link from 'next/link';
+
+export interface PricePerformanceProps
+  extends Partial<ButtonHTMLAttributes<any>>,
+    Record<string, any> {
+  /**
+   * Caption for the amount
+   */
+  amountSubtitle?: string;
+  /**
+   * The amount
+   */
+  amount?: string;
+  /**
+   * Performance
+   */
+  performance?: string;
+}
+
+const PerformanceHighlight = ({
+  amountSubtitle,
+  amount,
+  performance,
+  ...rest
+}: PricePerformanceProps) => {
+  return (
```

Comment:
Can we make sure that these are always full height to match each other?
<img width="738" alt="Screenshot 2021-02-13 at 09 05 40" src="https://user-images.githubusercontent.com/4254570/107846327-c62bf600-6dda-11eb-8800-f8e344c81482.png">